ow_power_system/src/power_system_node.cpp

```diff
@@ -102,6 +101,32 @@ void PowerSystemNode::powerCallback(const std_msgs::Float64::ConstPtr& msg)
   m_battery_temperature_pub.publish(battery_temperature_msg);
 }
 
+double PowerSystemNode::checkForFaults(double originalValue)
```

Comment:
It seems this function could be better named given how it is used, since it modifies the original mechanical power value.  Something like adjustMechPowerForFaults() or adjustForFaults() , etc.

Reply:
updated in #111